### PR TITLE
Minor improvements to surface diffusion

### PR DESCRIPTION
**Pull request by gassmoeller**
Follow up to #3407 with a few cosmetic improvements.

### CODE REVIEW

**Review by tjhei — APPROVED** [review]

File: benchmarks/diffusion_of_hill/analytical_topography.cc [hunk 1]
```diff
@@ -69,7 +69,11 @@ namespace aspect
       // loop over all of the surface cells and save the elevation to stored_value
       for (const auto &cell : this->get_triangulation().active_cell_iterators())
         if (cell->is_locally_owned() && cell->at_boundary())
+#if DEAL_II_VERSION_GTE(9,2,0)
```
Comment (tjhei) [inline benchmarks/diffusion_of_hill/analytical_topography.cc:72]: isn't this only for 9.3?

Comment (gassmoeller) [reply benchmarks/diffusion_of_hill/analytical_topography.cc]: You are right. Fixed now.

File: source/mesh_deformation/interface.cc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright (C) 2020 by the authors of the ASPECT code.
+  Copyright (C) 2014 - 2020 by the authors of the ASPECT code.
```
Comment (anne-glerum) [inline source/mesh_deformation/interface.cc:2]: Sorry for removing this! Originally it said `2011` instead of `2014`, is that important?

**Review by anne-glerum — APPROVED** [review]

File: source/mesh_deformation/interface.cc (hunk 2 above)
Comment (gassmoeller) [reply source/mesh_deformation/interface.cc:2]: No big deal, I think the 2011 was wrong as well, this file was added in 2014 (I checked the git log). Ready from my side.

**Comment by gassmoeller** [comment]
This seems to be approved.

**Pull request merged**